### PR TITLE
[WTF-1983] Update React Client Migration Guide

### PR DESCRIPTION
**Pull request by LEGIO-SEXTA-FERRATA**
This PR updates the migration part of the documentation for the new React based client. These changes should be released with 10.18 as it is the version when the React client switches from beta to generally available. 

Specifically:
- The widget migration sections are updated
- A new page is added for listing the automatic widget conversion limitations
- The batch conversion parts are decided to be removed altogether

### CODE REVIEW

File: content/en/docs/refguide/runtime/mendix-client/react.md [hunk 1]
```diff
@@ -185,3 +156,7 @@ JavaScript actions should be mostly unaffected by React. Problems arise only if
 Problems in JavaScript actions are not captured by the consistency checker. To validate that JavaScript actions work with the React client, the checker should be executed with the React client active. Any errors reported by the **Browser Console** (Press <kbd>F12</kbd> to access it) will help identify the problem.
 
 If you encounter problems with platform-supported JavaScript actions (for example from the [Nanoflow Commons](/appstore/modules/nanoflow-commons/) module) make sure to update the module containing the action from the Marketplace. In most cases, this will resolve the issue. If it is not resolved, reach out to the maintainer of the module and let them know of the issue.
+
+### Batch Conversion{#batch-conversion}
+
+We are working on a batch conversion component that lets you convert all incompatible widgets in your project with a few clicks. We will announce the release in our [Release Notes](/releasenotes/studio-pro/10/).
```
Comment (mlaponder) [inline content/en/docs/refguide/runtime/mendix-client/react.md]: iirc we decided not to mention this anymore in the docs

File: content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+description: "Describes the limitations of the automatic widget conversions."
+weight: 20
+---
+
+## Introduction
+
+Studio Pro, provides a quick and automatic method to easily migrate the widgets that are unsupported by the React client. However, not all the configuration options are currently supported by the replacement React client-ready widgets.
+
+{{% alert color="info" %}}
+We are working on a batch conversion component that lets you convert all incompatible widgets in your project with a few clicks. We will announce the release in our [Release Notes](/releasenotes/studio-pro/10/).
```
Comment (mlaponder) [inline content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md]: See previous comment

File: content/en/docs/refguide/runtime/mendix-client/react.md (hunk 1 above)
Comment (LEGIO-SEXTA-FERRATA) [reply content/en/docs/refguide/runtime/mendix-client/react.md]: Removed.

File: content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md (hunk 2 above)
Comment (LEGIO-SEXTA-FERRATA) [reply content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md]: Removed.

File: content/en/docs/refguide/runtime/mendix-client/react.md (hunk 1 above)
Comment (mlaponder) [reply content/en/docs/refguide/runtime/mendix-client/react.md]: ✨

File: content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md (hunk 2 above)
Comment (mlaponder) [reply content/en/docs/refguide/runtime/mendix-client/widget-conversion-limitations.md]: ✨

**Review by mlaponder — APPROVED** [review]

**Comment by LEGIO-SEXTA-FERRATA** [comment]
@ConnorLand, hello. We're done with the internal review. This PR is all yours! 😊

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**